### PR TITLE
Domain sample update and cleanup

### PR DESCRIPTION
**Pull request by doxiao**
1) Correct the envs in the domain-template to match the schema; moving them into serverPod element;
2) Remove check if the output dir is empty; clean the dir instead;
3) Remove unused functions;
4) Update the README files to be consistent with the new schema.

### CODE REVIEW

**Comment by doxiao** [comment]
The PR passed the integration test in Jenkins (http://wls-jenkins.us.oracle.com/job/weblogic-kubernetes-operator-javatest/668).

File: kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/README.md [hunk 1]
```diff
@@ -61,9 +61,9 @@ The PV and PVC creation inputs can be customized by editing the `create-pv-pvc-i
 
 ## Shared versus dedicated PVC
 
-By default, the `domainUID` is left empty in the inputs file so that the generated PV and PVC can be shared by multiple domain resources in the same Kubernetes namespaces.
+By default, the `domainUID` is left empty in the inputs file, which means the generated PV and PVC will not be tied to any particular domain. In other words, the generated PV and PVC can be shared by multiple domain resources in the same Kubernetes namespaces as the PV and PVC.
```
Comment (rosemarymarano) [inline kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/README.md]: will not be tied to any particular domain -> will not be associated with a particular domain, but can be shared by multiple domain resources in the same Kubernetes namespace as the PV and PVC.

**Comment by simon-meng-cn** [comment]
The change looks good to me.

**Comment by doxiao** [comment]
The latest version passed http://wls-jenkins.us.oracle.com/job/weblogic-kubernetes-operator-javatest/684/

**Pull request merged**